apps/site/lib/site_web/templates/layout/app.html.eex

```diff
@@ -10,9 +10,11 @@
     <% meta_description = assigns[:meta_description] || "Official website of the MBTA -- schedules, maps, and fare information for Greater Boston's public transportation system, including subway, commuter rail, bus routes, and boat lines." %>
     <meta name="description" content="<%= Phoenix.HTML.raw(meta_description) %>">
     <meta name="author" content="Massachusetts Bay Transportation Authority">
+    <%# Temporary verification to authenticate our domain for our Facebook account. This can be removed as soon as the verification process is complete. -- MSS 20200910 %>
+    <meta name="facebook-domain-verification" content="2jxewkdklzlgsdvn57f6s6gpa1cx0g" />
```

Comment:
So this verification will be removed later on?
If so, should we create a new Asana ticket for that?

Reply:
Yes, it is here: https://app.asana.com/0/385363666817452/1193278706153525